src/integrations/third-party/the-events-calendar.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package Yoast\WP\SEO\Integrations\Third_Party
+ */
+
+namespace Yoast\WP\SEO\Integrations\Third_Party;
+
+use Yoast\WP\SEO\Conditionals\Front_End_Conditional;
+use Yoast\WP\SEO\Conditionals\The_Events_Calendar_Conditional;
+use Yoast\WP\SEO\Conditionals\Open_Graph_Conditional;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+use Yoast\WP\SEO\Integrations\Third_Party\TheEventsCalendar\Schema;
+
+/**
+ * Class TheEventsCalendar
+ */
+class TheEventsCalendar implements Integration_Interface {
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_conditionals() {
+		return [ Front_End_Conditional::class, The_Events_Calendar_Conditional::class, Open_Graph_Conditional::class ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function register_hooks() {
+		add_filter( 'wpseo_schema_graph_pieces', array( $this, 'add_graph_pieces' ), 11, 2 );
```

Comment:
To avoid possible future namespace conflicts we try to prefix all function calls with `\` to ensure the function in the global namespace is used.
```suggestion
		\add_filter( 'wpseo_schema_graph_pieces', array( $this, 'add_graph_pieces' ), 11, 2 );
```